src/Modules/Contacts.php

```diff
@@ -77,7 +85,20 @@ public function emailStatement($id, $from = null, $to = null, $data = [])
             $params['end_date'] = $to->format('Y-m-d');
         }
 
-        $this->client->post($this->getUrl() . '/' . $id . '/statements/email', $data, $params);
+        if (in_array($filterBy, ['Status.Outstanding', 'Status.All'])) {
+            $params['filter_by'] = $filterBy;
+        }
+
+        //This apparently needs to be a string for zoho to accept it, so lets take a bool and make it a string
+        $params['send_unpaid_invoice_list'] = $sendUnpaidInvoiceList
```

Comment:
Do we know which one is the default? If the default is "neither", let's keep it that way so it's not a breaking change

Reply:
By default today the invoice list is not sent, so I've updated the default above as well.